File: Leonardo/Lista-complementar/desafio17.c

// Quantos DVD's tem a locadora
// Se o cliente atrasar: multa 10%
// 1/3 dos DVD's são alugados por mÊs
// 1/10 dos dvds são alugados
// 2% dos DVD's estragam por mÊs
//  e dos 2% 1/10 são repostos

#include <stdio.h>
int main(){
    float dvd, valor, multa, alugados, devolvidos, faturamento_mensal, faturamento_anual;

    printf("Digite a quantidade de DVD's: ");
    scanf("%f", &dvd);
    printf("Digite o valor do aluguel: ");
    scanf("%f", &valor);

    alugados = dvd / 3;
    devolvidos = alugados / 10;
    multa = alugados * 0.1;

    faturamento_anual = (alugados * valor * 12) + (devolvidos * multa * 12);

    printf("O faturamento anual é: %.1f", faturamento_anual);



    return 0;
}

File: Leonardo/Lista-complementar/desafio17.py
# Leitura da quantidade de DVDs e valor do aluguel
quantidade_dvds = int(input("Digite a quantidade de DVDs da locadora: "))
valor_aluguel = float(input("Digite o valor do aluguel: "))

# Cálculo do faturamento anual
dvds_alugados_mes = quantidade_dvds / 3
dvds_devolvidos_mes = dvds_alugados_mes / 10
multa_por_atraso = valor_aluguel * 0.1

faturamento_anual = (dvds_alugados_mes * valor_aluguel * 12) + (dvds_devolvidos_mes * multa_por_atraso * 12)

# Cálculo da quantidade total de DVDs após um ano
dvds_estragados_mes = quantidade_dvds * 0.02
dvds_repostos_mes = dvds_estragados_mes / 10

quantidade_dvds_ano = quantidade_dvds - (dvds_estragados_mes * 12) + (dvds_repostos_mes * 12)

# Exibição dos resultados
print("Faturamento anual da locadora: R$", faturamento_anual)
print("Quantidade total de DVDs após um ano:", quantidade_dvds_ano)
